### DISCUSSION TITLE
Date picker stuck on default value

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#5003] [Help] [answered]
Hello,

The date choice component returns to the default value even when the user changes it manually.

In my tests: 
With no default value, the component works properly. 
With a default value set to a specific fixed date, the component works properly. 
With a default value set to today : {{ now }}, the component doesn't update the value after a change or is reverting back to {{ now }} ?

https://user-images.githubusercontent.com/91942877/159485569-e20eea6c-5e4a-489a-87da-d9dd8a76b71f.mp4

Thank you!
Olivier

**Comment by melohagan** [comment, marked as answer]
Hi @diogenesbrussels 

Thanks for bringing this to our attention! 

I've verified the problem myself, and raised a bug for it here: https://github.com/Budibase/budibase/issues/5008
